docs/en/peripherals/dshot.md

```diff
@@ -147,3 +147,20 @@ ERROR [dshot] No data received. If telemetry is setup correctly, try again.
 
 Check manufacturer documentation for confirmation/details.
 :::
+
+
+## Bidirectional DShot
+
+Bidirectional DShot enables ESC RPM (eRPM) data collection from each ESC at high frequencies. This enhanced telemetry significantly improves the performance of [Dynamic Notch Filters](../config_mc/filter_tuning.md#dynamic-notch-filters) and enables more precise vehicle tuning.
+
+### Configuration
+
+To enable bidirectional DShot, set the [DSHOT_BIDIR_EN](../advanced_config/parameter_reference.md#DSHOT_BIDIR_EN) parameter.
+
+:::info
+The system calculates actual motor RPM from the received eRPM data using the [MOT_POLE_COUNT](../advanced_config/parameter_reference.md#MOT_POLE_COUNT) parameter. This parameter must be set correctly for accurate RPM reporting.
+:::info
+
+:::warning
+Bidirectional DShot is currently supported on H7 targets only, with support limited to 4 motor outputs.
```

Comment:
It's also supported on the i.MXRT (V6X-RT & Tropic) platforms up to 8 motor outputs.